### PR TITLE
feat(filter by level): add bracket group

### PR DESCRIPTION
**Pull request by Hacaw**
- created a new bracket group component to select the range level and update the chart-data accordingly
- created json of predicates to identify which bracket level is selected
- added prettier
- added git attributes to avoid line endings issues between different OS
- reduce chart-height to fit in 2k screen resolution and added bold font-weight
- updated readme
- formatted all files with prettier

### CODE REVIEW

**Comment by Hacaw** [comment]
@Repcan what do you think about these changes ? I cant add you to the reviewers, but I would like to see your feedback on these changes too, since you are now a contributor 👯‍♂️

**Comment by Repcan** [comment]
Looks good, suggestion for prettier settings:
    "tabWidth": 2,
    "trailingComma": "es5",
    "jsxBracketSameLine": true,
    "proseWrap": "never",
    "printWidth": 100

File: src/components/shared/Util.ts [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+  0: (player: any) => player, // no filter selected, default value
+  1: (player: any) => player.level < 20,
+  2: (player: any) => player.level > 20 && player.level < 30,
+  3: (player: any) => player.level > 30 && player.level < 40,
+  4: (player: any) => player.level > 40 && player.level < 50,
+  5: (player: any) => player.level > 50 && player.level < 60,
+  6: (player: any) => player.level > 60 && player.level < 70,
+  7: (player: any) => player.level > 70,
+};
+
```
Comment (Helias) [inline src/components/shared/Util.ts:14]: the level 20 is never read, 30 neither and so on
use <= or >= in some cases

```suggestion
const BRACKET_LEVEL_PREDICATES: { [key: number]: Function } = {
  0: (player: any) => player, // no filter selected, default value
  1: (player: any) => player.level < 20,
  2: (player: any) => player.level >= 20 && player.level < 30,
  3: (player: any) => player.level >= 30 && player.level < 40,
  4: (player: any) => player.level >= 40 && player.level < 50,
  5: (player: any) => player.level >= 50 && player.level < 60,
  6: (player: any) => player.level >= 60 && player.level < 70,
  7: (player: any) => player.level >= 70,
};

```

Comment (Helias) [reply src/components/shared/Util.ts:14]: we could even put a bracket with only 80 :P

**Comment by Hacaw** [comment]
> "tabWidth": 2,
> "trailingComma": "es5",
> "jsxBracketSameLine": true,
> "proseWrap": "never",
> "printWidth": 100

Thanks. I updated them. Please check now.

File: src/components/shared/Util.ts (hunk 1 above)
Comment (Hacaw) [reply src/components/shared/Util.ts]: Done. Please check them now.

**Comment by Hacaw** [comment]
> 0: (player: any) => player, // no filter selected, default value //show 1-80
> 1: (player: any) => player.level < 20, //show 1-19
> 2: (player: any) => player.level >= 20 && player.level < 30, // show 20-29
> 3: (player: any) => player.level >= 30 && player.level < 40, // show 30-39
> 4: (player: any) => player.level >= 40 && player.level < 50, // show 40-49
> 5: (player: any) => player.level >= 50 && player.level < 60, // show 50-59
> 6: (player: any) => player.level >= 60 && player.level < 70, // show 60-69
> 7: (player: any) => player.level >= 70 && player.level < 80, // show 70-79
> 8: (player: any) => player.level === 80, //show 80

Hehe, sorry, I didn't paid attention. Hope now it's ok.

**Review by Helias — APPROVED** [review]

**Pull request merged**